tests/mod.rs

```diff
@@ -54,16 +55,13 @@ fn assert_dirs_eq(expected: &Path, actual: &Path) {
     }
 
     // Ensure no unnecessary files were created
-    let walker = WalkDir::new(expected)
+    let walker = WalkDir::new(result)
         .into_iter()
         .filter_map(|e| e.ok())
         .filter(|e| e.file_type().is_file());
     for entry in walker {
-        let extra_file = entry
-            .path()
-            .strip_prefix(expected)
-            .expect("Comparison error");
-        let src_file = Path::new(actual).join(&extra_file);
+        let extra_file = entry.path().strip_prefix(result).expect("Comparison error");
+        let src_file = Path::new(expected).join(&extra_file);
```

Comment:
What was the intent of changing these variables around?

Reply:
I found the original name confusing, I forgot to rollback them, it was for me reasoning, I'll rollback this ^^' sorry